### PR TITLE
Saved palette feature

### PR DESCRIPTION
**Pull request by Marisa5280**
### Description
-This feature adds a side section for saved palettes.
-When a current palette is saved the mini color boxes will be added to the Saved palette section.
-If Saved Palette section is empty a message saying "No Saved Palettes" will display.

### Testing
-If the same palette is saved more than once only one will be saved and displayed.
-The data model is updated when the current palette is saved.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -71,26 +76,38 @@ function generateHexCode(){
...
+function dispayMinis(){
+    savedPaletteBoxCont.innerHTML += `
+    <div class="saved-palette-box-container">
+        <div class="mini-box" style="background-color: ${currentPalette.hexColors[0].hexCode}"></div>
+        <div class="mini-box" style="background-color: ${currentPalette.hexColors[1].hexCode}"></div>
+        <div class="mini-box" style="background-color: ${currentPalette.hexColors[2].hexCode}"></div>
+        <div class="mini-box" style="background-color: ${currentPalette.hexColors[3].hexCode}"></div>
+        <div class="mini-box" style="background-color: ${currentPalette.hexColors[4].hexCode}"></div>
+        </div>
+    `
```
Comment (caitlincradick) [inline main.js]: can we remove this space here ?

File: styles.css [hunk 2]
```diff
@@ -107,6 +115,41 @@ h1 {
...
+}
+
+.saved-palette-header {
+    text-align: center;
+    font-size: 30px;
+    margin-top: 40px;
+    font-weight: 600;
+}
+
 .hidden{
```
Comment (caitlincradick) [inline styles.css]: can we add a space after hidden ?

File: styles.css [hunk 3]
```diff
@@ -1,6 +1,14 @@
 * {
     font-family: 'Poppins', sans-serif;
     background-color: #E7F1F1;
+    /* border: 2px solid red  */
```
Comment (caitlincradick) [inline styles.css]: can we remove this please?

**Review by caitlincradick — CHANGES_REQUESTED** [review]
can we remove the spacing mentioned before merging, thanks !

File: main.js (hunk 1 above)
Comment (Marisa5280) [reply main.js]: like move the back tick on line 108 to 107?

File: styles.css (hunk 3 above)
Comment (Marisa5280) [reply styles.css]: will do

File: styles.css (hunk 2 above)
Comment (Marisa5280) [reply styles.css]: yep!

**Pull request merged**